README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
+```sh
+brew install go-task
+```
```

Comment:
The command 'task dev' should include a brief description of what it does, especially for users unfamiliar with the project context. Explain that it runs the server and builds assets, ideally in watch mode.